### ISSUE TITLE
Don't nuke package.json when doing ng init

### ISSUE THREAD

**Issue by zlepper** [angular/angular-cli#3059]
Sorry for not using your form, but it doesn't really fit with a feature request.

As far as I have understood, to update an app with angular-cli, we have to run `ng init` in the root folder. Doing this updates the app as intended. However during the lines of questions, you will be asked if you want to replace package.json. As package.json contains all the dependencies for the app at the current state, these change have to be merged in manually. 
I would like to be able to have the cli to this automatically, so i don't risk losing all my current dependencies. 

So in short, the cli should only update dependencies it knows about, and not touch the rest. 

If i knew where to start, i would be happy to try and implement it into the cli itself, but i'm not quite sure were i would start.

**Comment by filipesilva** [comment]
The mechanism for `ng init` is a simple diff, it doesn't go much further than that. So currently this isn't a request we can accommodate, but we're looking at having a better update mechanism than `ng init` in the future.
